src/components/loader.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+const Loader = () => {
+  return (
+    <>
+      <div className="flex flex-col h-screen justify-center items-center">
+        <div className="">
```

Comment:
why empty string. the whole div can be removed. right ?

Reply:
addressed